ThePensionsRegulator.Frontend/Styles/_tpr-header-menu.scss

```diff
@@ -0,0 +1,339 @@
+@import '_tpr-variables.scss';
+@import 'govuk/base';
+
+.nav-open_overlay {
+    display: none;
+}
+
+.tpr-mobile-menu-toggle {
+    display: flex;
+    justify-content: flex-end;
+    background-color: $tpr-colour-tolopea;
+    padding: 2rem;
+    flex-direction: row;
+    margin-left: 4rem;
+    padding-right: 0.5rem;
+    padding-left: 0.5rem;
+    margin-top: -0.5rem;
+    margin-bottom: -0.5rem;
+
+    &:focus {
+        outline: $govuk-focus-width solid $govuk-focus-colour;
+    }
+}
+
+.closed {
+    display: inline;
+    padding-left: 0.6rem;
+    background: 0 0;
+    border: none;
+    color: $tpr-colour-white;
+    font-size: 1.1rem;
+    font-family: $govuk-font-family;
+}
+
+.open {
+    background-color: $tpr-colour-whisper;
+    align-items: baseline;
+    padding: 1.8rem;
+    padding-right: 0.5rem;
+    padding-left: 1rem;
+}
+
+.open::before {
+    content: "x";
+    font-size: 1.5rem;
+    font-weight: 700;
+    font-family: $govuk-font-family;
+    color: $tpr-colour-royal-blue;
+}
+
+.tpr-mobile-menu__svg {
+    width: 1.5rem;
+    height: 1.5rem;
+    transform: scaleX(-1);
+    filter: FlipH;
+    vertical-align: bottom;
+    display: flex;
+}
+
+.tpr-mobile-menu__svg.hide {
+    display: none;
+}
+
+.tpr-mobile-menu-toggle .si-glyph-fill {
+    fill: $tpr-colour-white;
+}
+
+.tpr-header-menu-nav {
+    display: none;
+}
+
+%mobile-menu-visible {
```

Comment:
Using the % sign like this is a neat trick in SASS that I wasn't aware of, but since it's only used once you could just move this implementation to where it's used and simplify it a bit.